doc/source/reference/random/generator.rst

```diff
@@ -71,17 +71,17 @@ By default, `Generator.permuted` returns a copy.  To operate in-place with
 `Generator.permuted`, pass the same array as the first argument *and* as
 the value of the ``out`` parameter.  For example,
 
-    >>> rng = np.random.default_rng()
+    >>> rng = np.random.default_rng(12345)
     >>> x = np.arange(0, 15).reshape(3, 5)
     >>> x
     array([[ 0,  1,  2,  3,  4],
            [ 5,  6,  7,  8,  9],
            [10, 11, 12, 13, 14]])
     >>> y = rng.permuted(x, axis=1, out=x)
     >>> x
-    array([[ 1,  0,  2,  4,  3],  # random
```

Comment:
Please put back `# random` (even if you change the output to match the seed set in `conftest.py`), it's helpful here because the output of the example should indeed be random.

Reply:
Also done.